palvelutarjotin/settings.py

```diff
@@ -353,3 +356,8 @@
 NOTIFICATIONS_IMPORTER = (
     "notification_importers.notification_importer.NotificationFileImporter"
 )
+
+UPDATE_LAST_LOGIN = {
+    "ENABLED": env.bool("UPDATE_LAST_LOGIN"),
+    "UPDATE_INTERVAL": env.int("UPDATE_LAST_LOGIN_INTERVAL_MINUTES"),
+}
```

Comment:
The environment variable names UPDATE_LAST_LOGIN and UPDATE_LAST_LOGIN_INTERVAL_MINUTES seem transparent to me so why not just use them with direct mapping to settings? i.e.:
```python
UPDATE_LAST_LOGIN = env.bool("UPDATE_LAST_LOGIN")
UPDATE_LAST_LOGIN_INTERVAL_MINUTES = env.int("UPDATE_LAST_LOGIN_INTERVAL_MINUTES")
```

UPDATE_LAST_LOGIN["UPDATE_INTERVAL"] drops the MINUTES from the variable name and thus makes it less clear what is the unit. One could argue in favor of keeping the MINUTES in the variable name here with e.g. Zen of Python's "Explicit is better than implicit".

Reply:
Did a change